<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement2">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal2"/>
   </item>
   <item row="0" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal1"/>
   </item>
   <item row="1" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement2">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal3"/>
   </item>
   <item row="3" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal4"/>
   </item>
   <item row="4" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal5"/>
   </item>
   <item row="5" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal6"/>
   </item>
   <item row="6" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal7"/>
   </item>
   <item row="7" column="2">
    <widget class="TaurusLabel" name="taurusLabelVal8"/>
   </item>
   <item row="3" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement4">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement7">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement3">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement5">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement5">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement6">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement8">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement7">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement8">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement3">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement1">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="TaurusValueComboBox" name="sourceMeasurement4">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement1">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="TaurusValueComboBox" name="configMeasurement6">
     <property name="autoApply">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusLabel</class>
   <extends>QLabel</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueComboBox</class>
   <extends>QComboBox</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
